packages/schematics/angular/class/schema.json

```diff
@@ -12,20 +12,11 @@
       "type": "string",
       "format": "path",
       "description": "The path to create the class.",
-      "default": "app",
       "visible": false
     },
-    "sourceDir": {
```

Comment:
Removing options is a breaking change, we will need a new minor on everything. Not that we didn't before, this isn't the first change... but I just want to highlight that.